src/doc/dev_manual/RCDevelopment.rst

```diff
@@ -83,6 +83,12 @@ and locally. ::
 Apply the same changes to develop
 ---------------------------------
 
+Typically, the same changes applied to the release candidate then also need to be applied to ``develop``.
+This is not always the case however.
+Some changes are made only for the release candidate and should never get applied to ``develop``.
+If this is the case, make sure that the PR to the release candidate includes a comment, either in the initial PR comment or in a comment at the end of the conversation on the PR, that the work is not to be applied to ``develop``.
```

Comment:
We might just wanna restrict ourselves to a comment at the end of the conversation. Then, we always know where to go look for it. We could also decide to adopt a convention in the specific format of the comment so that we can ensure quality searching too.

Reply:
I believe having it in a standard location would be a good idea, so my preference is to add it in a comment at the end. That's also where the comment indicating it was merged to develop will be located. It looks to me like all RC PRs will now have comment at the end indicating it wasn't merged or it was merged with the PR number. Should this be stated explicitely somewhere?